plugins/outdated.yaml

```
@@ -43,6 +43,14 @@ spec:
     bin: outdated.exe
   shortDescription: Finds outdated container images running in a cluster
   homepage: https://github.com/replicatedhq/outdated
+  caveats: |
+    Usage:
+      $ kubectl outdated
+
+    For additional options:
+      $ kubectl outdated --help
+      or https://github.com/replicatedhq/outdated/blob/v0.4.0/doc/USAGE.md
```

```suggestion
      or https://github.com/replicatedhq/outdated/blob/master/doc/USAGE.md
```
Please use a stable link here. I understand your urge to link to the correct docs for this exact version. But it will mean that version bumps don't get auto-approved by our automation.